Expound on field query param for the data json format and geojson format (#2085)
onaio/onadata: docs/data.rst

      ]

Sort form data against a column(field)
----------------------------------------
Returns submitted data for a specific form in a sorted order. Use the `sort` parameter to filter the list of submissions.

> The sort parameter is takes a field(form column) as the key and select value pairs as demonstrated below.

Reviewer: I think the previous version was clear enough... I don't see the need to change it

        "subscriberid": "639027...60317"
    }

Fetch data on select columns for a given form
---------------------------------------------------

> Queries select column data specified by the user. Use the ``fields`` parameter to specify the column data that should be returned.

Reviewer: ```suggestion
Returns a list of the selected columns from the submitted data. Use the ``fields`` parameter....
```
